test/unit/Elsa.UnitTests/Models/VariablesTests.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using Xunit;
+
+namespace Elsa.Models
+{
+    public class VariablesTests
+    {
+        [Theory(DisplayName = "The Remove method should remove a variable if such a variable exists"), AutoMoqData]
+        public void Remove_removes_a_variable_by_name_if_present(string variableName, object variableData)
```

Comment:
I could do without the underscores in the test name, and simply use e.g. Test1. The DisplayName is a neat way of describing what the test is all about.